.github/workflows/updatecli.yaml

```diff
@@ -23,13 +29,7 @@ jobs:
           flags: "--config ./updatecli/updatecli.d --values ./updatecli/values.github-action.yaml"
         env:
           # Github "generated" token is used because the repository secrets are not available in PRs.
-          UPDATECLI_GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-      - uses: tibdex/github-app-token@v1.5
-        id: generate_token
-        if: github.ref == 'refs/heads/main'
-        with:
-          app_id: ${{ secrets.JENKINS_ADMIN_APP_ID }}
-          private_key: ${{ secrets.JENKINS_ADMIN_APP_PRIVKEY }}
```

Comment:
Removing this blocs will fail the "apply" stage (https://github.com/jenkins-infra/status/pull/185/files#diff-f3c1fa6e77c6c2264c2ed7ac4c908d4baee1aca7f4d2582914ec08138be8088cR41)